crates/egui-winit/src/lib.rs

```diff
@@ -746,15 +746,19 @@ impl State {
             physical_key
         );
 
-        if let Some(logical_key) = logical_key {
+        // "Logical OR physical key" is a fallback mechanism for keyboard layouts without Latin characters: it lets them
+        // emit events as if the corresponding keys from the Latin layout were pressed. In this case, clipboard shortcuts
+        // are mapped to the physical keys that normally contain C, X, V, etc.
+        // See also: https://github.com/emilk/egui/issues/3653
```

Comment:
I assume you have tested this?

I'm a bit surprised that `logical_key` is a `None` in these cases.

Reply:
yup — I myself have a Russian keyboard layout which prompted #4081 in the first place unfortunately